### DISCUSSION TITLE
Any ways to skip both the audio processing and the transcription??

### DISCUSSION THREAD

**Discussion by Ibrahima-prog** [R3gm/SoniTranslate#111] [Q&A]
Hello i already have the SRT file, are there any ways to skip both the audio processing and the transcription part and directly jump into the translation part and continue from there?? I do not need any of them as the only thing i want is to add translated subtitles to the video, nothing to do with the audio at all.
I already uploaded the srt but audio processing still happens. 

### **The part Below**

[INFO] >> Vocal Track Isolation and Voiceless Track Separation...
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 721/721 [07:15<00:00,  1.66it/s]
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 721/721 [10:25<00:00,  1.15it/s]
[INFO] >> Vocal Clarity Enhancement through De-Reverberation...
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 353/353 [1:23:00<00:00, 14.11s/it]
100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 353/353 [1:24:06<00:00, 14.30s/it]

**Comment by biraescudero** [comment]
Yes! The interface has an element where you can upload the subtitles file.

Em seg., 28 de out. de 2024 10:19, Ibrahima Sory CAMARA <
***@***.***> escreveu:

**Reply by Ibrahima-prog** [reply]
Really sorry i actually i meant both the audio processing and the transcription. I do not need any of them as the only thing i want is to add translated subtitles to the video, nothing to do with the audio at all.
I already uploaded the srt but audio processing still happens. Thank you for your response though, i appreciate it. (i have edited my question)
